src/sequential_cursor.rs

```diff
@@ -72,7 +76,7 @@ impl<'s> std::io::Seek for SeqCursor<'s> {
                     .cursors
                     .iter()
                     .fold(0u64, |acc, cursor| acc + cursor.get_ref().len() as u64);
-                (total as i64 - rel) as u64
+                (total as i64 + rel) as u64
```

Comment:
Thank you! The notion of `FromEnd` was apparently not clear to me at the time of impl. I double checked in this is correct CC @dralley

Reply:
@drahnr @TommyLike As this transformed from `-` to `+`, the `as i64` shouldn't be necessary any longer

Reply:
I take it back, might depend on the variant of  `SeekFrom` https://doc.rust-lang.org/stable/std/io/enum.SeekFrom.html